var/spack/repos/builtin/packages/damaris/package.py

```diff
@@ -13,80 +13,72 @@ class Damaris(CMakePackage):
 
     homepage = "https://project.inria.fr/damaris/"
     git = "https://gitlab.inria.fr/Damaris/damaris.git"
-    maintainers("jcbowden")
-
-    version("master", branch="master")
-    version("1.6.0", tag="v1.6.0")
-    version("1.5.0", tag="v1.5.0")
-    version("1.3.3", tag="v1.3.3")
-    version("1.3.2", tag="v1.3.2")
-    version("1.3.1", tag="v1.3.1")
```

Comment:
Historically versions are deprecated for at least a release before being removed.  See https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions.

Reply:
Thanks for the pointers, it has been a while since I have updated this package.